### DISCUSSION TITLE
Multiple MAC address with the same lease/ip

### DISCUSSION THREAD

**Discussion by ackstorm23** [TechnitiumSoftware/DnsServer#716] [Q&A]
When using other DHCP software,  I was able to assign multiple MAC addresses to the same lease so that they would receive the same IP.

This was useful for devices which may switch between wireless or wired connections on the same subnet.  

Is there a way to do this with DnsServer?  So far that I've played with it, I do not see a way to do this which has forced me to use bonding configurations with the local docking station's eth0 and wireless radio's wlan0 devices (forcing the same MAC address) in order to accomplish this behavior, which is too advanced of functionality for many devices.

**Comment by ShreyasZare** [comment]
Thanks for asking. There is no option currently to do this. I will plan to add support for it in later release. 

I would suggest that you post this in the Issues section so that it gets tracked and you will know when the feature is available.
